content/how-to-configure-a-cabal-project-for-profiling/index.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "How to configure a Cabal project for profiling"
+date = 2024-11-27
+[taxonomies]
+authors = ["Hécate"]
+categories = ["Cabal"]
+tags = ["Performance", "Profiling"]
++++
+
+The [Cabal Manual][Cabal Manual] now has a guide on how to analyse the performance of Haskell applications.
```

Comment:
I still disagree with "analyze the performance of Haskell applications". My version: 
```suggestion
The [Cabal Manual][Cabal Manual] now has a guide on how to enable profiling in Haskell projects.
```

Reply:
Let's have this conversation in https://github.com/haskell/cabal/pull/10606 first.